src/main/java/com/locallampoon/fiveh/core/UserInput.java

```diff
@@ -6,9 +6,9 @@
 
 class UserInput {
 
-    private static final List<String> ACTIONS = new ArrayList<>(Arrays.asList("fight", "go", "flee" , "move", "get", "grab", "drop",
-            "talk", "inspect", "h", "help", "i", "inventory", "q", "quit", "fight", "attack", "hit", "punch", "kick",
-            "flee", "run", "recruit"));
+    private static final List<String> ACTIONS = new ArrayList<>(Arrays.asList(
```

Comment:
Feels odd to have a list of ACTIONS here but also a switch case elsewhere... these things should be one and the same, and referenced accordingly. Maybe for a future PR?

Reply:
That's a good point. If all these commands are already listed in the switch statement, it may be redundant for checking invalid commands when it can be covered in the switch's default statement.